diff -u a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2023-06-27 14:43:48.721120000 +1000
+++ b/CMakeLists.txt	2023-06-27 14:44:19.375112000 +1000
@@ -135,6 +135,7 @@
 ################################################################################
 
 # prepares a tar.gz of the sources and/or binaries
+install(DIRECTORY ${CMAKE_Fortran_MODULE_DIRECTORY}/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} )
 ecbuild_install_project( NAME fv3jedilm )
 
 # print the summary of the configuration
